src/index.js

```diff
@@ -1,8 +1,12 @@
 import React from 'react';
 import { render } from 'react-dom';
 import App from './components/App/App';
+import { Provider } from 'react-redux';
 
```

Comment:
you'll want to import store here and then everything will render on your front-end. 

`import store from './store';`